CHANGELOG.md

```diff
@@ -1,301 +1,119 @@
 # Change log
 
-The `next` version of 4.29 is now available. Planned release date is February 2024.
+The `next` version of 4.30 is now available. Planned release date is June 2024.
 
-![Current build version](https://img.shields.io/npm/v/arcgis-js-api/next?label=Current%20build)
-
-## Performance updates
-
-The [heatmapRendererCreator.createRenderer()](https://developers.arcgis.com/javascript/latest/api-reference/esri-smartMapping-renderers-heatmap.html#createRenderer) method for generating a [HeatmapRenderer](https://developers.arcgis.com/javascript/latest/api-reference/esri-renderers-HeatmapRenderer.html) is 100x faster than it was in version 4.28.
+![Current build version](https://img.shields.io/npm/v/@arcgis/core/next?label=Current%20build)
 
 ## Layer updates
```

Comment:
I would probably just remove this or put TBD if it's going to be empty

Reply:
Thanks, I added TBD.